docs: apply alert style correctly at warning message (#424)
actions/javascript-action: README.md

   npm run all
   ```

> [!WARNING]
>
> This step is important! It will run [`ncc`](https://github.com/vercel/ncc) to
> build the final JavaScript action code with all dependencies included. If you
> do not run this step, your action will not work correctly when it is used in a
> workflow. This step also includes the `--license` option for `ncc`, which will
> create a license file for all of the production node modules used in your

> > project.

Reviewer: Might be easier to simply remove the warning for now. Looks like moving it causes ordered list issues.

> 5. Commit your changes

Reviewer: ```suggestion
1. Commit your changes
```

Author: [An alert markdown](https://github.com/orgs/community/discussions/16925) didn't work within other elements.
Then we would like to use reorder list.

x-ref: [Discussion comment](https://github.com/orgs/community/discussions/16925#discussioncomment-10195289)

   ```bash
   git add .
   git commit -m "My first action is ready!"
   ```

> 6. Push them to your repository

Reviewer: ```suggestion
1. Push them to your repository
```

   ```bash
   git push -u origin releases/v1
   ```

> 7. Create a pull request and get feedback on your action
> 8. Merge the pull request into the `main` branch

Reviewer: ```suggestion
1. Create a pull request and get feedback on your action
1. Merge the pull request into the `main` branch
```
